shopify/utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .exceptions import *
+from .utilities import SessionTokenUtility
```

Comment:
I'm -1 on "utils", "utilities", "tools", "extra", "misc", "helpers", etc.

It's not a hill I always want to die on, but I think a name that helps clarify what should go in and what shouldn't is always nicer. In this case, maybe `session_tokens`?

Also, the exceptions aren't too cumbersome. It would simplify things if we just kept everything in `session_tokens.py`.